### ISSUE TITLE
Dots like a horizontal scroll bar

### ISSUE THREAD

**Issue by vangn** [metafizzy/flickity#537]
Hi, first of all, it's a great library, I used it in my projects, but I wanted to ask if it's possible to make or simulate the points (flickity-page-dots) act as a horizontal scroll bar like the options draggable and freeScroll?

Thanks and greetings from Mexico

**Comment by desandro** [comment]
Thanks for this feature request. No this is not currently a feature of Flickity. **Add a 👍  reaction** if you would like to see this feature added. Do not add +1 comments, [they will be deleted](http://metafizzy.co/blog/use-github-reactions-delete-plus-1-comments/).

**Comment by tractanz** [comment]
@desandro thanks so much for the great library, I use it in a lot of my projects!

Draggable dots/slide navigation would be an amazing feature! Would it be possible to sync a Flickity slider with a jQuery UI Slider (https://jqueryui.com/slider/) or something similar?

**Comment by desandro** [comment]
> Would it be possible to sync a Flickity slider with a jQuery UI Slider (https://jqueryui.com/slider/) or something similar?

Yes, this is possible via the [Flickity API](https://flickity.metafizzy.co/api.html)

**Comment by yepecece** [comment]
Does anyone as an exemple for this?

Thank you.

**Comment by tractanz** [comment]
> Does anyone has an exemple for this?
> (draggable slider as nav)
> 
> Thank you.

Hi yepecece, unfortunately I couldn't get working in time. I required a draggable scrollbar and ended up using swiper slider which has builtin scrollbar - https://swiperjs.com/swiper-api#scrollbar.

Though if you require a Flickity carousel to be navigation for another Flickity carousel, checkout the asNavFor section on the Flickity options page - https://flickity.metafizzy.co/options.html

Cheers,
